.github/workflows/update-staging-version.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow triggers a new workflow inside the graasp-deploy repository. It passes a json 
+# with the repository name and the latest tag pushed from the caller repository. 
+name: Push new tag to graasp-deploy repository
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push tag events.
+  push:
+    tags:
+      - v*
```

Comment:
Is there a stricter pattern we can use here for Semver? @dialexo @pyphilia @victorggonzalez ?

Reply:
I found this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
In particular, we could use something like `v[0-9]+.[0-9]+.[0-9]+` or similar (depending on what version ranges we would like to support).